### PR TITLE
Refactor wiki documentation

### PR DESCRIPTION
**Pull request by e-pettersson-ericsson**
### Applicable Issues
Wiki documentation was hard to navigate, and across the bundles the information was inconsistent. Tutorials for GitLab and Cx bundles were hidden/mixed with other information.

### Description of the Change
Major refactoring of wiki documentation

* Moved section about client set up from Getting_Started.md to Hosting_Environment.md
* Made a directory for each bundle and split up the set up from tutorials and overview pictures. Easier to extend with more bundles in the future...
* Added links from Bundles.md to each bundle markdown files
* Fixed some spelling mistakes and shuffled around information so the bundles are aligned in structure.
* Removed unneccessary markdown directory (_it looks like some Getting_Started.md + Hosting_Environments.md were removed but it's just Git not recognizing the move .._)
* No information has been added/removed, just moved around

### Benefits
Separation of tutorials and installation/startup guides for each bundle.  Aligning information across bundles.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Emelie Pettersson emelie.pettersson@ericsson.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,6 @@ This repository forms part of the Eiffel Community. Eiffel is a protocol for tec
 Visit [Eiffel Community](https://eiffel-community.github.io) to get started and get involved.
 
 # Documentation
-1. [**Hosting Environments**](./wiki/markdown/Hosting_Environments.md)
-2. [**Getting Started**](./wiki/markdown/Getting_Started.md)
-3. [**Bundles**](./wiki/markdown/Bundles.md)
+1. [**Hosting Environments**](wiki/Hosting_Environments.md)
+2. [**Getting Started**](wiki/Getting_Started.md)
+3. [**Bundles**](wiki/Bundles.md)
```
Comment (e-backmark-ericsson) [inline README.md]: Shouldn't these links be prepended with './'?

Comment (e-pettersson-ericsson) [reply README.md]: Yes they should, my editor is messing with me ...

Comment (e-backmark-ericsson) [reply README.md]: Good!
You didn't change anything in the bundles docs or tutorials other than some structure, right? Not that you're not allowed to, but I believe it is best to first restructure and then update text in a new PR if needed.

Comment (e-pettersson-ericsson) [reply README.md]: Yes, just moved around existing information and fixed some spelling mistakes. Nothing new and nothing removed (despite what Git diff shows)

Comment (e-backmark-ericsson) [reply README.md]: Approved.
@tobiasake , please clarify the Helm init command documentation in a new PR if you see the need for it.

**Pull request merged**